### PR TITLE
Set up SNS

### LINKED ISSUE #130: Make SNS script
## Description
Write a python script to send SNS message for new genres

## Required Files
- alert.py

## User Story
As a user I need regular updates so I can stay in the loop with the latest releases.

### PR DESCRIPTION
**Pull request by annalisev**
Emails are now working for our arbitrary tags.

Closes #130 .

### CODE REVIEW

File: pipeline/notification/alert.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+'''A script that sends out SNS alerts when new games come in.'''
+
+from os import environ as ENV
+
+from dotenv import load_dotenv
```
Comment (hhm970) [inline pipeline/notification/alert.py:5]: Make sure to remove this and anything `dotenv` related when Dockerising this

File: pipeline/notification/alert.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+def send_sns(topic: str, games: list, config):
+    '''Sends a SNS message to subscribers of a given topic
+    about newly released games.'''
+
+    sns = client(
+        "sns", aws_access_key_id=config["AWS_KEY"], aws_secret_access_key=config["AWS_SECRET"])
+    sns.publish(
+        TargetArn=TAG_ARNS[topic],
+        Message=f'''Hey there!
```
Comment (hhm970) [inline pipeline/notification/alert.py:22]: To clean this up, you could use `\n` instead of actual newlines?

File: pipeline/notification/alert.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+from dotenv import load_dotenv
+from boto3 import client
+
+TAG_ARNS = {'Action': 'arn:aws:sns:eu-west-2:129033205317:c10-games-action-tag',
+            'Adventure': 'arn:aws:sns:eu-west-2:129033205317:c10-games-adventure-tag',
+            'Indie': 'arn:aws:sns:eu-west-2:129033205317:c10-games-indie-tag',
+            'Horror': 'arn:aws:sns:eu-west-2:129033205317:c10-games-horror-tag'}
+
+
+def send_sns(topic: str, games: list, config):
```
Comment (hhm970) [inline pipeline/notification/alert.py:14]: Make sure to put `-> None`

File: pipeline/notification/alert.py (hunk 2 above)
Comment (annalisev) [reply pipeline/notification/alert.py:22]: I did try to! But the publish functionality doesn't like backslash characters unfortunately. I will see if there is another way around this.

File: pipeline/notification/alert.py (hunk 3 above)
Comment (annalisev) [inline pipeline/notification/alert.py:14]:
```suggestion
def send_sns(topic: str, games: list, config) -> None:
```

**Review by hhm970 — APPROVED** [review]
Looks good, I can't see any discrepancies! Thanks for taking my feedback on board :)

**Pull request merged**